wiki/Help_centre/Account_restrictions/en.md

```diff
@@ -103,13 +111,15 @@ Please understand that sending an email to this address only guarantees a second
 
 ## Common restriction reasons and cooldowns
 
-| Restriction reason | Cooldown duration | Appeal notes |
+| Restriction reason | Cooldown duration (minimum) | Appeal notes |
 | :-- | :-- | :-- |
 | Judgement error | No cooldown | If found erroneous, no impact on the account. |
 | Multi-account | Permanent (multis) | Main account will be left unrestricted, possibly silenced. |
 | Excessive multi-accounting | 3 months | Appeal terms apply, 1 year tournament ban |
 | Account sharing | 3 months | Appeal terms apply, 1 year tournament ban, and partial rollback |
-| Cheating | 3 months | Appeal terms apply, 1 year tournament ban, and full rollback |
+| Cheating | 6 months | Appeal terms apply, 1 year tournament ban, and full rollback |
 | Excessive misconduct | Variable | Appeal terms apply. Cooldown will depend on history. |
-| Cheating/sharing for a tournament | 6 months | Appeal terms apply, appropriate rollback, permanent tournament ban |
-| Subsequent offences | 6 months | Appeal terms apply, appropriate rollback, tournament ban 1 year per offence |
+| Cheating/sharing for a tournament | 12 months | Appeal terms apply, appropriate rollback, permanent tournament ban |
+| Subsequent offences (multi-accounting, account sharing, community conduct) | 6 months | Appeal terms apply, rollback (where appropriate), tournament ban 1 year per offence |
+| Subsequent offences (cheating) | 12 months, up to 24 | Appeal terms apply, rollback, tournament ban 1 year per offence |
```

Comment:
Just to make sure I understand, repeatedly cheating and returning would give you a maximum of 24 months of cooldown, but any other offense can have the cooldown escalate indefinitely?

Reply:
Subsequent offences cap at 6 months for non-cheating offenses, as described. For cheating offenses, they double from 6 to 12 to 24 and then everything after that is 24.

Reply:
Well what's this "minimum" in this table for if it doesn't apply to most of this table? Subsequent offences for non-cheating offences would always result 6 months of cooldown (meaning it's not really a minimum). Where does it apply? Is it possible to get more than 3 months of cooldown for (the first offence of) account sharing?

Reply:
I've removed the mention of minimum from the table header and also re-ordered it to be smallest to largest in terms of applied cooldown.